setup.py

```diff
@@ -9,15 +9,21 @@
     readme = readme_file.read()
 
 requirements = [
+    "aicspylibczi==2.4.1",
+    "dask>=2.9.0",
+    "distributed>=2.9.3",
     "numpy>=1.16",
     "imageio>=2.3.0",
-    "scipy>=1.1.0",
-    "tifffile==0.15.1",
+    "lxml>=4.4.2",
+    "tifffile>=2019.7.26.2",
```

Comment:
AWESOME! b'bye 0.15.1

Reply:
I remember this was causing issues with reading LZW compressed CZIs and that's why we stuck on 0.15.1 😮 did that get resolved?

Reply:
Seems so? I tested a random selection of 300 production tiffs and no issues.